common/src/main/scala/uk/ac/wellcome/models/UnifiedItem.scala

```diff
@@ -0,0 +1,15 @@
+package uk.ac.wellcome.models
+
+import com.sksamuel.elastic4s._
+import uk.ac.wellcome.utils.JsonUtil
+
+
+case class UnifiedItem(
```

Comment:
Is this the "CleanedRecord" class we were working with last week?

What’s the convention for documenting/commenting classes in Scala?